pages/library/_viewId.vue

```diff
@@ -55,6 +64,12 @@ import Vue from 'vue';
 import { chunk } from 'lodash';
 import { BaseItemDto } from '~/api/api';
 
+interface FilterItem {
```

Comment:
You should reuse the type from other component. Avoid duplicating types, it leads to errors. Feel free to make a `types` folder in the root of the repository to store them, if needed.